use-cases/plutus-use-cases.cabal

```diff
@@ -0,0 +1,36 @@
+cabal-version: 2.0
+name: plutus-use-cases

```

Comment:
Can we keep the directory name the same as the package name unless there's a good reason not to?